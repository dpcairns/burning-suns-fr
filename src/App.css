.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}

button {
  background-color: #6772e5;
  color: #fff;
  box-shadow: 0 4px 6px rgba(50, 50, 93, 0.11), 0 1px 3px rgba(0, 0, 0, 0.08);
  padding: 7px 14px;
}
button:hover {
  background-color: #5469d4;
}

.App-link {
  color: #61dafb;
}

@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}


form {
  margin: 1em;

}

h4 {
  font-size: 2em;
  font-weight: bolder;
  color: red;
}

h3 {
  color: black;

}

.MuiFormControl-fullWidth{
  margin: 1em;
}

  .paper {
    margin-top: 2em;
    display: flex;
    flex-direction: column;
    align-items: center;
    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
  }
  .avatar {
    margin: 2em;
    background-color:rgb(23, 163, 223);
    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
  }
  .form {
    width: '100%'; 
    margin-top: 2em;
    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
  }

  .submit {
    margin: spacing(3, 0, 2),
  }

