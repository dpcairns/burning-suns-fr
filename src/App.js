import React, {Component} from "react";
import {
  BrowserRouter as Router,
  Switch,
  Route,
  Link
} from "react-router-dom"
import SignInPage from './signInPage';
import SignUpPage from './signUpPage';
import DetailPage from './DetailPage';
import SearchPage from './SearchPage';

import { render } from "@testing-library/react";

export default class App extends Component{

  handleUserChange = (newToken) => {
    this.setState({ token: newToken })
   
  }
  render() {
    return (
      <Router>
      
        <Switch>
          <Route path="/SignInPage">
            <SignInPage handleUserChange={this.handleUserChange} />
          </Route>
          <Route path="/SignUpPage">
            <SignUpPage />
          </Route> 
          <Route path="/SearchPage">
            <SearchPage />
          </Route>
          <Route path="/DetailPage">
            <DetailPage />
          </Route>
         </Switch>
    </Router>
  );
}
}
