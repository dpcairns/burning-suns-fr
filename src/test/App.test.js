import React from 'react';
import { shallow } from 'enzyme';
import App  from '../App'


describe('First React component test with Enzyme', () => {
    it('renders without crashing', () => {
      const wrapper = shallow(<App />);
      expect(wrapper).toMatchSnapshot()
     });
 });
 