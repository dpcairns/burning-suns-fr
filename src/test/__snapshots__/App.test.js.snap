// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`First React component test with Enzyme renders without crashing 1`] = `
<BrowserRouter>
  <Header
    handleUserChange={[Function]}
    token={null}
  />
  <Switch>
    <Route
      exact={true}
      path="/"
      render={[Function]}
    />
    <Route
      path="/SignInPage"
      render={[Function]}
    />
    <Route
      path="/SignUpPage"
      render={[Function]}
    />
    <Route
      path="/Calendar"
      render={[Function]}
    />
    <PrivateRoute
      path="/SearchPage"
      render={[Function]}
      token={null}
    />
    <PrivateRoute
      path="/DetailPage/:city"
      render={[Function]}
      token={null}
    />
    <PrivateRoute
      path="/StarredListPage"
      render={[Function]}
      token={null}
    />
    <PrivateRoute
      path="/Notes"
      render={[Function]}
      token={null}
    />
    <Route
      path="/aboutUs"
      render={[Function]}
    />
  </Switch>
  <Copyright />
</BrowserRouter>
`;
